cms/static/js/views/utils/xblock_utils.js

```diff
@@ -73,13 +73,23 @@ define(["jquery", "underscore", "gettext", "common/js/components/utils/view_util
         deleteXBlock = function(xblockInfo, xblockType) {
             var deletion = $.Deferred(),
                 url = ModuleUtils.getUpdateUrl(xblockInfo.id),
-                xblockType = xblockType || gettext('component');
+                additionalHeaderText = '',
+                additionalBodyText = '';
+            xblockType = xblockType || gettext('component');
+            if (xblockInfo.get('is_prereq')) {
+                additionalHeaderText = gettext(' (and prerequisite)');
```

Comment:
You can't concatenate together text like this-- it simply won't work for translation. You need to use interpolation and placeholders.